styles.css

```diff
@@ -380,8 +384,81 @@ li {
   background: #606bc3;
 }
 
+.modal_footer {
+  display: flex;
+  flex-direction: column;
+}
+
+.section_modal {
+  display: none;
+}
+
+.show {
+  z-index: 10;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  inset: 0;
+  backdrop-filter: blur(5px);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.modal {
+  background-color: white;
+  position: relative;
+  width: 90%;
+  height: 95%;
+  max-width: 500px;
+  border-radius: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  overflow: scroll;
+  margin: 15px;
+}
+
+.modal_content {
+  padding: 5%;
+}
+
+.header_modal {
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.title_modal {
+  font-size: x-large;
+  color: #172b4d;
+}
+
+.modal_buttons {
+  display: flex;
+  justify-content: center;
+  gap: 5%;
+}
+
+.button_modal {
+  display: flex;
+  font-size: medium;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+}
```

Comment:
[Optional] Kindly consider adding a `cursor: pointer` to your buttons so users hover on/around a button they can know it's clickable to create a good user experience.

It would be nice if you can apply this in your project and not just for buttons but also icons and anything clickable in your project.

Reply:
thank you @Strangeal I made the changes.